include/rocksdb/utilities/backupable_db.h

```diff
@@ -175,8 +206,9 @@ struct BackupableDBOptions {
       uint64_t _restore_rate_limit = 0, int _max_background_operations = 1,
       uint64_t _callback_trigger_interval_size = 4 * 1024 * 1024,
       int _max_valid_backups_to_open = INT_MAX,
-      BackupTableNameOption _share_files_with_checksum_naming =
-          kOptionalChecksumAndDbSessionId)
+      ShareFilesNaming _share_files_with_checksum_naming =
+          static_cast<ShareFilesNaming>(kUseDbSessionId | kFlagIncludeFileSize |
+                                        kFlagMatchInterimNaming))
```

Comment:
We should tell the internal user to disable `kFlagMatchInterimNaming` right? (Assuming their script continues relying on size always being present.)

Reply:
It's up to them which hurts more: names that break their workflow or space associated with file duplication. We're making them opt-in to file duplication, because that's almost irreversible if invoked by accident. And of course having kFlagMatchInterimNaming in the default is harmless for people without early-6.12 SST files.